src/content/concepts/dependency-graph.md

```diff
@@ -16,4 +16,10 @@ Any time one file depends on another, webpack treats this as a _dependency_. Thi
 When webpack processes your application, it starts from a list of modules defined on the command line or in its configuration file.
 Starting from these [_entry points_](/concepts/entry-points/), webpack recursively builds a _dependency graph_ that includes every module your application needs, then bundles all of those modules into a small number of _bundles_ - often, just one - to be loaded by the browser.
 
+Sometimes it is very helpful to visualize the dependency graph using tools like `webpack-visualizer`. You can install it by:
+
+``` bash
+npm install webpack-visualizer-plugin
```

Comment:
The latest version was published [4 years ago](https://www.npmjs.com/package/webpack-visualizer-plugin), and it won't work with webpack 5 https://github.com/chrisbateman/webpack-visualizer/issues/71.